src/binutil/server.rs

```diff
@@ -56,7 +56,7 @@ pub fn run_tikv(mut config: TiKvConfig) {
     if let Err(e) = config.validate() {
         fatal!("invalid configuration: {}", e.description());
     }
-    info!(
+    debug!(
```

Comment:
Logging configuration is important. It help diagnose problems a lot. Sometimes we just say, talk is cheap, show me your configuration in logs.